File: app/src/main/java/com/ianfield/bodyscoring/models/Score.java
package com.ianfield.bodyscoring.models;

import java.util.Locale;

import io.realm.RealmObject;

/**
 * Created by Ian on 14/01/2016.
 */
public class Score extends RealmObject {

    double score;

    int count = 0;

    public double getScore() {
        return score;
    }

    public void setScore(double score) {
        this.score = score;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public String toString() {
        return String.format(Locale.getDefault(), "Score: %.1f, Count: %d", score, count);
    }
}

File: app/src/main/java/com/ianfield/bodyscoring/models/Score.kt
package com.ianfield.bodyscoring.models

import java.util.Locale

import io.realm.RealmObject

/**
 * Created by Ian on 14/01/2016.
 */
open class Score : RealmObject() {

    var score: Double = 0.toDouble()

    var count = 0

    override fun toString(): String {
        return String.format(Locale.getDefault(), "Score: %.1f, Count: %d", score, count)
    }
}
